release/_publishing.sh

```diff
@@ -1,5 +1,84 @@
 #!/bin/bash
 
+function add_fixed_issues_to_project_version {
+	IFS=',' read -r -a fixed_issues_array < "${_BUILD_DIR}/release/release-notes.txt"
+
+	local fixed_issues_array_length="${#fixed_issues_array[@]}"
+
+	local fixed_issues_array_part_length=$((fixed_issues_array_length / 4))
+
+	for counter in {0..3}
+	do
+		local number_of_issues_added=$((counter * fixed_issues_array_part_length))
+
+		if [ "${counter}" -eq 3 ]
+		then
+			fixed_issues_array_part_length=$((fixed_issues_array_length - number_of_issues_added))
+		fi
+
+		IFS=',' fixed_issues="${fixed_issues_array[*]:number_of_issues_added:fixed_issues_array_part_length}"
+
+		if (curl \
+			"https://patcher.liferay.com/api/jsonws/osb-patcher-portlet.project_versions/updateFixedIssues" \
+			--data-raw "fixedIssues=${fixed_issues}&patcherProjectVersionId=${1}" \
+			--fail \
+			--max-time 10 \
+			--output /dev/null \
+			--retry 3 \
+			--silent \
+			--user "${LIFERAY_RELEASE_PATCHER_PORTAL_EMAIL}:${LIFERAY_RELEASE_PATCHER_PORTAL_PASSWORD}")
+		then
+			lc_log INFO "Adding fixed issues to the ${_PRODUCT_VERSION} project version."
+		else
+			lc_log ERROR "Unable to add the full fixed issues list to the ${_PRODUCT_VERSION} project version."
+
+			return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+		fi
+	done
+
+	lc_log INFO "The full fixed issues list has been added to the ${_PRODUCT_VERSION} project version."
+}
+
+function add_patcher_project_version {
+	local product_version_label="Quarterly Releases"
+
+	local root_patcher_project_version_name=""
+
+	case "${_PRODUCT_VERSION}" in
+	7.4.*)
+		product_version_label="DXP 7.4"
+
+		root_patcher_project_version_name="7.4.13-ga1"
+		;;
+	7.3.*)
+		product_version_label="DXP 7.3"
+
+		root_patcher_project_version_name="fix-pack-base-7310"
+		;;
+	esac
+
+	local add_by_name_response=$(\
+		curl \
+			"https://patcher.liferay.com/api/jsonws/osb-patcher-portlet.project_versions/addByName" \
+			--data-raw "combinedBranch=true&committish=${_PRODUCT_VERSION}&fixedIssues=&name=${_PRODUCT_VERSION}&productVersionLabel=${product_version_label}&repositoryName=liferay-portal-ee&rootPatcherProjectVersionName=${root_patcher_project_version_name}" \
```

Comment:
@kiwm, can you please try to perform a better indentation as the other `curl` parameters?